sentences/ro/light_HassLightSet.yaml

```diff
@@ -6,6 +6,7 @@ intents:
           - "[<seteaza>] <luminozitatea> [<din>] <name> [la] [valoarea [de]] <brightness>"
           - "[<seteaza>] <name> [la] <luminozitatea> [de] <brightness>"
           - "[<seteaza>] [<in> | la] <brightness> <luminozitatea> [<din>] <name>"
+          - "[<seteaza>] <name> [la] [valoarea [de]] <brightness>"
```

Comment:
In theory, saying "plafoniera hol 30" would trigger this sentence. Maybe we should at least have another brightness placeholder with `(la sut(ă|a) | % | [de] procente)` mandatory instead of optional?

@gabimarchidan any thoughts?